src/runtime/test_case_runner.js

```diff
@@ -127,17 +127,15 @@ export default class TestCaseRunner {
 
   shouldUpdateStatus(testStepResult) {
     switch (testStepResult.status) {
+      case Status.UNDEFINED:
```

Comment:
Is this change needed to keep the tests passing after the change below?

If we are making this change, I'd prefer we also update the return for these three cases to be
```javascript
return (
  this.result.status !== Status.FAILED ||
  this.result.status !== Status.AMBIGUOUS || 
  this.result.status !== Status.UNDEFINED 
)
```
It would probably also be good to refactor this to use lodash's [_.some](https://lodash.com/docs/4.17.11#some)

Reply:
@charlierudolph